Cookbook/Technical-Documents/Current-guide.md

```diff
@@ -0,0 +1,36 @@
+# `Current` Guide - how to use it in Babylon iOS Project
+
+### Where can I find it?
+- `Current` is the global instance of the `World` type which is located in the `BabylonDependencies.framework`. 
+Its purpose is to provide convenient access to the instances that should be shared between different parts of the app (cross-cutting concerns). For more information please read the [How Control the World](/Cookbook/Proposals/ControlTheWorld.md) document.
+
+## `Do`s and `Don't`s
+
+#### Do
+- access `Current` **only** from topmost `AppDelegate` until `Builder`s layer
+- pass the required dependencies down the chain;
+- create a customized version to fit your specific needs **for tests** and set it in the test `setUp` method. Please reset it on the `tearDown` method 
```

Comment:
I think this is no longer the right approach since we have started to run unit tests in parallel. Am I right?

Reply:
We have parallel **and** serial tests running. Imagine the tests `a`, `b`, and `c` in a process and `d`, `e` and `f` in another.. 

When `a` finishes, it's important to reset `Current` in `tearDown`. 
@TheAdamBorek ^
@AliSoftware 
@ilyapuchka

Reply:
I confirm what @daniloaliberti said, that's also my understanding of how they are distributed across sims. Parallel testing doesn't mean you shouldn't reset the state.